### PR TITLE
Show the entity name as pill

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: components/search/SearchForm.vue [hunk 1]
```diff
@@ -3,32 +3,54 @@
...
+      <template
+        v-if="pillLabel"
+        v-slot:prepend
       >
-    </b-button>
+        <SearchBarPill
+          :text="pillLabel"
+          :remove-link-label="$t('removeEntityFilter', { entityLabel: pillLabel })"
+          :remove-link-to="{
+            path: localePath({
```
Comment (lbiedinger) [inline components/search/SearchForm.vue]: instead of having multiline code in the template here, should this be a computed property or method?

Comment (rwd) [reply components/search/SearchForm.vue]: Agreed, let's do this already.

File: components/search/SearchForm.vue [hunk 2]
```diff
@@ -3,32 +3,54 @@
...
-        :alt="$t('search')"
+    <b-input-group>
+      <template
+        v-if="pillLabel"
+        v-slot:prepend
       >
-    </b-button>
+        <SearchBarPill
+          :text="pillLabel"
+          :remove-link-label="$t('removeEntityFilter', { entityLabel: pillLabel })"
```
Comment (rwd) [inline components/search/SearchForm.vue]: This should be supplied from the store too. The search form component should not need to know anything about entities.

Comment (rwd) [reply components/search/SearchForm.vue]: On second thoughts, all that should be needed is to rename the translation and placeholder in lang/en.js as its content is entity-agnostic anyway.

So change it in lang/en.js to: `"removeFilter": "Remove {filterLabel} filter"`

**Pull request merged**